wiki/Shortcut_key_reference/nl.md

```diff
@@ -0,0 +1,290 @@
+---
+tags:
+  - hotkey
+  - hotkeys
+  - keyboard
+  - shortcuts
+---
+
+# Sneltoets combinaties
+
+Osu! heeft een groot aantal sneltoets combinaties die kunnnen worden gebruikt in-game of wanneer je op een bepaalt scherm bent. De combinaties en hun functies zijn hieronder vermeld.
```

Comment:
Osu! heeft een groot aantal sneltoetscombinaties die je in-game kunt gebruiken of wanneer je op een bepaalt scherm bent.